activerecord/lib/active_record/core.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
 require "active_support/core_ext/enumerable"
-require "active_support/core_ext/hash/indifferent_access"
 require "active_support/core_ext/string/filters"
 require "active_support/parameter_filter"
 require "concurrent/map"
 
 module ActiveRecord
   module Core
     extend ActiveSupport::Concern
+    include ActiveModel::Access
```

Comment:
In my original revision, I put this line in `ActiveRecord::Base`.  However, `ActiveRecord::Base` doesn't seem like an appropriate place to put the `:call-seq:` comments, and I felt like this line and the `:call-seq:` comments should be in the same file.